bump_rpm.sh

```diff
@@ -55,6 +55,15 @@ else
 	NEW_VERSION=$2
 fi
 
+if [[ -n $GITHUB_ACTION ]] ; then
+	if [[ -n $GEM_NAME ]] ; then
+		echo "::set-output name=gem_name::$GEM_NAME"
+	fi
+	echo "::set-output name=package_name::$PACKAGE_NAME"
+	echo "::set-output name=current_version::$CURRENT_VERSION"
+	echo "::set-output name=new_version::$NEW_VERSION"
+fi
```

Comment:
In the end I didn't need this. Should I remove this?

Reply:
weeeell I guess it might be usefull, but we get the same info in the matrix, right?
Tho I believe that if you have an output to share, you should and this might be interesting in the future :shrug: